astropy/table/table.py

```diff
@@ -1521,6 +1521,40 @@ def _repr_html_(self):
     def __repr__(self):
         return self._base_repr_(html=False, max_width=None)
 
+    def _repr_latex_ (self, latexdict={}):
+        """
+        Format Table object into the LaTeX format and print to the screen.
+
+        Parameters
+        ----------
+        latexdict : str or dict
+            Dictionary of LaTeX formatting options or either one of these three
+            strings corresponding to pre-defined options, ``AA``, ``doublelines``,
+            ``template``. Defaults to ``{}``.
+
+        Notes
+        -----
+        Uses ``latex.Latex.write()`` function from ``astropy.io.ascii`` to change
```

Comment:
```suggestion
        Uses `astropy.io.ascii.latex.Latex.write` function to change
```
If you give the full name in single backticks, then sphinx will turn that int oa link when the documentation is build. That way, you can click on it and get more information.